### DISCUSSION TITLE
Is there any update to becoming a retrieval provider?

### DISCUSSION THREAD

**Discussion by LoganC034** [filecoin-project/community#544] [Network + Implementations Announcements]
Is there any update to becoming a retrieval provider? Can it be done today? What is the relation to the myel project?

**Comment by rjan90** [comment]
Hey @LoganC034! Yeah, you can check out https://strn.network/ and https://www.notion.so/pl-strflt/Filecoin-Saturn-efc122f123f344ff8ff0de6071954dba

**Reply by LoganC034** [reply]
This is exactly what i was looking for. Ill check it out!

**Reply by LoganC034** [reply]
@rjan90 Im going to work on setting up on the testnet today. Thanks for the assist. 

I have an older-ish server Im setting up on and from reading sounds like I would meet all of the requirements for a production/private beta node w/ the exception of a 10GB/s connection. However, looks like I would far exceed the proposed Station LY2. I wonder when I could begin to test that product also?

My specs are the following : 
 
Linux
Dual CPUs => Intel Xeon CPU E5-2670 V2 (2.5Ghz 10 Cores each)
224 GB ECC RAM
2TB NVMe SSD in a RAID 1 Config
8TB SAS (10GB/s) in a RAID 5 Config
Network is 400MB/s Upload 400MB/s Download

Looks like I could upgrade the network to 1 or even 2GB/s but not really sure if that makes sense economically. Ill have to do research on what the Saturn/FIL rewards system looks like.

I see there is a Slack channel for Saturn, is there also a discord?

**Reply by rjan90** [reply]
> I see there is a Slack channel for Saturn, is there also a discord?

Yeah, there’s a link to their Slack-channel at the bottom of the Saturn-website I think, it should be #filecoin-saturn

**Reply by guglez** [reply]
That link is not working. If you click it it says that you are not a member (It looks like there isn’t an account on Filecoin tied to this email address.) I'm running the Saturn node but not getting any payments. That's pretty frustrating because it's very expensive to operate a node with 10G uplink.

**Reply by TippyFlitsUK** [reply]
Are you in discussion with the Saturn team on Slack @guglez?
